libraries/ui-library/package.json

```diff
@@ -46,11 +46,11 @@
     "@stencil/react-output-target": "^0.5.3",
     "@stencil/sass": "^3.0.7",
     "@stencil/vue-output-target": "^0.8.7",
-    "@types/jest": "27.0.3",
+    "@types/jest": "^29.5.11",
     "@typescript-eslint/eslint-plugin": "^6.15.0",
     "@typescript-eslint/parser": "^6.15.0",
-    "jest": "27.4.5",
-    "jest-cli": "27.4.5",
+    "jest": "^29.7.0",
```

Comment:
It looks like Stencil still uses Jest 27.x internally, did you check if they made some upgrading efforts? 

Otherwise we have to postpone that upgrade. There is a dependency internally to the Jest version they used from what I remember. Unless they decoupled that in the meantime...

Reply:
Jest 29 is supported: https://stenciljs.com/docs/support-policy#testing-libraries